schema.sql

```diff
@@ -5,5 +5,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL(5, 2)
+    weight_kg DECIMAL(5, 2),
+    species VARCHAR(255)
```

Comment:
- [ ] Altering the table directly is not recommended, you should use the "ALTER TABLE" query in this case.